File: app/src/main/java/siergo_o/onlinernews/model/RssFeed.java
package siergo_o.onlinernews.model;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root (name = "rss", strict = false)
public class RssFeed {

    @Element(name = "channel")
    public RssNewsChannel channel;


    @Override
    public String toString() {
        return "RssFeed [channel=" + channel + "]";
    }

    public RssNewsChannel getChannel() {
        return channel;
    }

    public void setChannel(RssNewsChannel channel) {
        this.channel = channel;
    }
}

File: app/src/main/java/siergo_o/onlinernews/model/RssFeed.kt
package siergo_o.onlinernews.model;

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "rss", strict = false)

class RssFeed {
    @field:Element(name = "channel")
    var channel: RssNewsChannel? = null

    override fun toString(): String {
        return "RssFeed [channel=$channel]"
    }
}

